docs/sql-guide/functions/function-coalesce.md

```diff
@@ -0,0 +1,116 @@
+---
+title: COALESCE()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# COALESCE() function
+
+`COALESCE()` function is used to return the first non-null value among the provided parameters.
+
+## Syntax
+
+```
+coalesce(value1, value2, ...)
+```
+
+## Arguments
+
+| Argument | Description | Length |
+|---|---|---|---|
+| `value1`,`value2`.. | Any expressions or values of same data type. The function returns the first non-null value among them. | 1 |
+
+## Returns
+
+The data type and value of the first non-null parameter
```

Comment:
Phrasing is odd here because the examples are returning values from the tables, not a data type.

How about something like:
"...Returns each value from the table column listed in the function, where there is a null value on the same row..."

Reply:
Another viewpoint: in the examples a SELECT...COALESCE() query returns:
* multiple values
* allows substitution of a value for the `null`
So the phrasing you're using doesn't appear correct? But I'm not an expert :D